caniusepython3/__main__.py

```diff
@@ -46,11 +47,13 @@ def projects_from_cli(args):
                         help='verbose output (e.g. list compatibility overrides)')
     parser.add_argument('--exclude', '-e', action='append', default=[],
                         help='Ignore list')
+    parser.add_argument('--index', '-i', default=dependencies.INDEX_URL,
+                        help=index_help)
     parsed = parser.parse_args(args)
-
     if not (parsed.requirements or parsed.metadata or parsed.projects):
         parser.error("Missing 'requirements', 'metadata', or 'projects'")
 
+    dependencies.INDEX_URL = parsed.index
```

Comment:
I don't know if I love mutating a module global in order to influence what index is used. Since `check()` is the only public API for this package there shouldn't be a problem in changing other internal APIs to pass this in. It also might be time to restructure this code to not have all CLI parsing here to just lead to the list of projects.

Reply:
I agree - explicit argument passing makes much more sense.